3rdparty/libcam/libcam_encoder/encoder.c

```diff
@@ -449,11 +449,15 @@ static encoder_video_context_t *encoder_video_init(encoder_context_t *encoder_ct
 		return (enc_video_ctx);
 	}
 
+#if LIBAVCODEC_VER_AT_LEAST(53,6)
     video_codec_data->codec_context = getLoadLibsInstance()->m_avcodec_alloc_context3(video_codec_data->codec);
-
-    getLoadLibsInstance()->m_avcodec_get_context_defaults3 (
-			video_codec_data->codec_context,
-			video_codec_data->codec);
+    #if !LIBAVCODEC_VER_AT_LEAST(60,0)
+        getLoadLibsInstance()->m_avcodec_get_context_defaults3(video_codec_data->codec_context, video_codec_data->codec);
```

Comment:
in ffmpeg6.0 ，If you need to make an accident for the variable of the function, it is recommended that the parameters are linked below.

https://lists.ffmpeg.org/pipermail/ffmpeg-cvslog/2016-June/100801.html

Reply:
I don't quite understand what you mean. Do I need to replace `avcodec_get_context_defaults3` to `init_context_defaults` in ffmpeg 6.0?